lib/cli/commands-schema/common-options/global.js

```diff
@@ -5,6 +5,10 @@ module.exports = {
   version: { usage: 'Show version info', shortcut: 'v', type: 'boolean' },
   verbose: { usage: 'Show verbose logs', type: 'boolean' },
   debug: { usage: 'Namespace of debug logs to expose (use "*" to display all)', type: 'string' },
+  param: {
```

Comment:
This is a service specific option, so should rather be defined in `service.js`

Reply:
great point, fixed